plugins/lua/logistics.lua

```diff
@@ -6,94 +6,176 @@ local overlay = require('plugins.overlay')
 local utils = require('utils')
 local widgets = require('gui.widgets')
 
-local function make_stat(name, stockpile_number, stats, configs)
-    return {
-        enabled=configs[stockpile_number] and configs[stockpile_number][name] == 'true',
-        designated=stats[name..'_designated'][stockpile_number] or 0,
-        can_designate=stats[name..'_can_designate'][stockpile_number] or 0,
+-------------------------------------
+-- logistics status console output
+--
+
+local function make_stat(type, sp_number, stats, configs)
+    local stat = {
+        enabled=(configs[sp_number] ~= nil) and (configs[sp_number][type] == 'true'),
```

Comment:
unless you want to specifically include `false` as a valid value, it's better to leave off the `~= nil`

Reply:
Yeah, I'm depending on enabled to become true or false here as a result of nil was messing up my logic in lines like this
```
has_claim_enabled = has_claim_enabled or sp.claim.enabled
```

However, I wasn't sure testing for `nil` like am here is common practice in lua or if there was some other shorthand method.